crypto/keyring_streaming_test.go

```diff
@@ -0,0 +1,228 @@
+package crypto
+
+import (
+	"bytes"
+	"io"
+	"testing"
+
+	"github.com/pkg/errors"
+)
+
+const testFilename = "filename.txt"
+
+func TestKeyRing_EncryptStream(t *testing.T) {
```

Comment:
I'd also test reading the encrypted data and decrypting with the "normal" functions to prevent any kind of regression (even though I know they're now implemented using the streaming API under the hood)

Reply:
True, I'll add that

Reply:
Done